server/lib/convertTime.js

```diff
@@ -0,0 +1,13 @@
+const convertTime = (number) => {
```

Comment:
@stevkim future refactor - lets handle conversion of timeSlot reservations times on backend